.github/workflows/clingo.yml

```diff
@@ -68,6 +73,11 @@ jobs:
           
           git clone https://github.com/spack/spack.git spack-src
           . spack-src/share/spack/setup-env.sh
+          if [[ ${{ matrix.python-version }} == "3.12.0" ]] ; then
+            cd spack-src              
+            wget -q -O - https://github.com/spack/spack/commit/1452e927719ddbbf07118e88f9fd8d18fabf0fae.patch | git apply -v 
```

Comment:
Feels a bit awkward to be me that this works by accident. Can you add a note saying that this only works because setuptools is already installed as a system package? So, the wrong setuptools *.pth file runs :(